clusters/veritable-prod/alice/ui/values.yaml

```diff
@@ -25,7 +23,14 @@ postgresql:
 ingress:
   hostname: alice.vrtbl.com
   ingressClassName: nginx-alice
+emailTransport: SMTP_EMAIL
+smtpHost: smtp.azurecomm.net
+smtpPort: 587
+emailFromAddress: "alice@vrtbl.com"
+emailAdminAddress: "alice@vrtbl.com"
+smtpSecure: true
 smtpCredentials:
   enabled: true
-  userSecret: blah
-  passwordSecret: blah
+  existingSecret: smtp-credentials
+  existingSecretUserKey: smtp_username
+  existingSecretPasswordKey: smtp_client_secret
```

Comment:
Where were these created?

Reply:
These are created in terraform.  I have not yet done a PR for that as I need to test it.

Reply:
ok gotcha, just checking you hadn't forgotten secrets here